src/os/bluestore/BlueStore.cc

```diff
@@ -7668,9 +7668,19 @@ int BlueStore::_is_bluefs(bool create, bool* ret)
 * opens both DB and dependant super_meta, FreelistManager and allocator
 * in the proper order
 */
-int BlueStore::_open_db_and_around(bool read_only, bool to_repair)
-{
-  dout(5) << __func__ << "::NCB::read_only=" << read_only << ", to_repair=" << to_repair << dendl;
+int BlueStore::_open_db_and_around(
+  bool read_only,
+  bool to_repair,
+  bool apply_deferred,
+  bool remove_deferred)
```

Comment:
Not a must for this specific PR, but can we create types for these 4 boolean values? having a function receive 4 boolean params is an invitation for problems...